src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml

```diff
@@ -701,6 +701,12 @@
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <!-- Uno workaround: On Android, the Height of ContentPresenters inside this Grid is not bing calculated correctly -->
+                            <!-- It can be removed once this PR is merged: https://github.com/unoplatform/uno/pull/18261 -->
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
```

Comment:
I tried to reproduce this behavior outside of the TabView context inside an isolated test, but I was not able to do it.